modules/acs-cloud-responsibilities.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cloud_service/acs-cloud-responsibility-matrix.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-cloud-responsibilities_{context}"]
+= Shared responsibilities for {product-title-managed-short}
+
+While Red Hat manages the {product-title-managed-short} services, the customer has certain responsibilities, as shown in the following table.
+
+[cols="3,1",options="header"]
+|===
+
+|Resource
+|Responsibility
+
+|Platform monitoring |Red Hat
+
+a|Hosted components:
```

Comment:
I'm assuming that "Hosted components" refers to the Central services, is that correct?

If that's true, I think that we're also responsible for high availability, uptime, and backups, just as for the Control Plane (which I'm assuming refers to fleet manager - or, from a customer point of view, the ability to provision and deprovision ACS instances).

Reply:
Regarding the first question: that's also how I understand the doc: it's the one _hosted on our side_.

@kcarmichael08 I agree that it is also not entirely clear to me wht "Control plane" really refers to. At first I understood it so that it refers to Central UI, which acts as some sort of control plane for the secured clusters. But then, how would that differ from "Hosted components"?